tendrl/notifier/objects/alert/__init__.py

```diff
@@ -0,0 +1,48 @@
+from tendrl.commons import objects
```

Comment:
Why this file and a few others in this PR showing up as a new file it should clearly reflect that the file is being refactored from a different path right?? Can you just check if you have deleted this file in the old path and commited the delete..

Is this really a new code??

I would suggest you break this commit based on the purpose of the commit like:
1. Refactoring
2. New Changes
etc..